sklearn/linear_model/_coordinate_descent.py

```diff
@@ -22,9 +22,9 @@
 from ..model_selection import check_cv
 from ..utils.extmath import safe_sparse_dot
 from ..utils.fixes import _astype_copy_false, _joblib_parallel_args
-from ..utils.validation import check_is_fitted, _check_sample_weight
-from ..utils.validation import column_or_1d
-from ..utils.validation import _deprecate_positional_args
+from ..utils.validation import (check_consistent_length, check_is_fitted,
+                                _check_sample_weight, column_or_1d,
+                                _deprecate_positional_args)
```

Comment:
Since imports get modified, probably using black code formatting style is suitable for future modifications?
```suggestion
from ..utils.validation import (
    check_consistent_length,
    check_is_fitted,
    _check_sample_weight,
    column_or_1d,
    _deprecate_positional_args,
)
```